examples/applications/mock-vas/src/chariott_provider.rs

```diff
@@ -6,6 +6,13 @@ use std::sync::Arc;
 use std::vec;
 
 use async_trait::async_trait;
+use chariott_proto::{
+    common::{
+        discover_fulfillment::Service, DiscoverFulfillment, FulfillmentEnum, FulfillmentMessage,
+        IntentEnum, InvokeFulfillment, Value as ProtoValue,
```

Comment:
Do we not want to import and use `ValueMessage` here instead of aliasing `Value` as `ProtoValue`?

Reply:
Perhaps some more cases to review?

    $ git grep -E '\bas\s+Proto'
    examples/common/src/chariott/api.rs:        discover_fulfillment::Service as ProtoService, DiscoverFulfillment, DiscoverIntent,
    examples/common/src/chariott/inspection.rs:    fulfillment::Fulfillment, inspect_fulfillment::Entry as ProtoEntry, InspectFulfillment,
    examples/common/src/examples/detection.rs:    DetectEntry, DetectRequest as ProtoDetectRequest, DetectResponse as ProtoDetectResponse,

Reply:
Done in 44ce11f2310210263841d7709e36adec95269785.